### PR TITLE
Rename instances of Multisignal Renewal to PyRenew

### PR DESCRIPTION
**Pull request by damonbayer**

### CODE REVIEW

**Review by dylanhmorris — APPROVED** [review]
LGTM

**Comment by damonbayer** [comment]
> LGTM, pending fix to CodeCov upload.

Also need to fix the dependency check action

File: docs/source/developer_documentation.rst
Comment (damonbayer) [inline docs/source/developer_documentation.rst:1]: `PyRenew-devs` group does not exist

**Comment by dylanhmorris** [comment]
> > LGTM, pending fix to CodeCov upload.
> 
> Also need to fix the dependency check action

Fixing this in #400

File: docs/source/developer_documentation.rst
Comment (damonbayer) [reply docs/source/developer_documentation.rst:1]: @dylanhmorris are you able to rename the group or make a new group?

Comment (dylanhmorris) [reply docs/source/developer_documentation.rst:1]: Done https://github.com/orgs/CDCgov/teams/pyrenew-devs

**Pull request merged**